chatty-cathy-client/src/main/java/org/client/Client.java

```diff
@@ -0,0 +1,70 @@
+package org.client;
+
+import org.common.Message;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.*;
+import java.net.*;
+
+public class Client {
+
+    private static final Logger logger = LoggerFactory.getLogger(Client.class);
+    private ObjectInputStream serverIn = null;
+
+    public Client(String addr, int port) throws IOException {
+        BufferedReader in;
+        ObjectOutputStream out;
+
+        Socket s;
```

Comment:
This is used very far away from its declaration so should be more descriptively named


```suggestion
        Socket socket;
```